scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -802,6 +802,12 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) { parser =>
     def startTokenPos = in.tokenPos
     def endTokenPos = in.prevTokenPos
   }
+
+  case object EmptyPos extends Pos {
+    val startTokenPos = in.tokenPos
+    val endTokenPos = in.tokenPos - 1
```

Comment:
wouldn't the position be empty if `start == end` (since `end` is excluded)?

Reply:
I think it's only empty if the end is before the start. That happens with autoPos if ther is nothing to parse for the body.